glossary.yml

```diff
@@ -907,8 +922,9 @@
   en:
     term: "catch (an exception)"
     def: >
-      To accept responsibility for handling an error or other unexpected event. R
-      prefers "handling a condition" to "catching an exception".
+      To accept responsibility for handling an error or other unexpected event. [R](#r_language)
+      prefers "handling a condition" to "catching an exception". [Python](#python), on the other hand,
+      encourages raising and catching exceptions, and in some situations, requires it.
```

Comment:
In truth, "condition handling" is something I have never heard of in programming. But my background is in Python/Java, and I actively avoid R. That sentence was already there, and since I don't know nuances in R, I left it alone and just explained the preferred way in Python.

I did google 'condition handling R' and it mostly sounds the same as what I would call exception handling. I have found reference to both `conditions` and `exceptions` in R, but I don't see how they inherently differ (but they must, right?).